arb_os/arbowner.mini

```diff
@@ -535,6 +499,63 @@ impure func arbowner_setL1GasPriceEstimate(_topFrame: EvmCallFrame, calldata: By
     evmOp_return(0, 0);
 }
 
+impure func arbowner_setFairGasPriceSender(_topFrame: EvmCallFrame, calldata: ByteArray) { // (address)
```

Comment:
This was just moved around in this PR, but this comment is wrong. The params are `(address, bool)`.